testsuite/features/secondary/trad_baremetal_discovery.feature

```diff
@@ -23,8 +23,10 @@ Feature: Bare metal discovery
     And the PXE default profile should be enabled
 
   Scenario: Register a client for bare metal discovery
-    When I register using "1-spacewalk-bootstrap-activation-key" key
-    Then I should see "sle_client" in spacewalk
+    Given I am authorized as "admin" with password "admin"
+    When I bootstrap traditional client "sle_client" using bootstrap script with activation key "1-spacewalk-bootstrap-activation-key" from the proxy
```

Comment:
If we remove `I should see "sle_client" in spacewalk`everywhere, then maybe we should also remove it from the step definitions?

Reply:
Let's check :-)

Reply:
I replaced this step definition with solution suing `spacecmd` command instead.

Reply:
Thanks